average.js

```diff
@@ -1,6 +1,76 @@
+/*function findFilterOutliers(numbers) { 
```

Comment:
Do not leave commented code in the commit. What is the purpose of this code?

Reply:
I coded this and commented this to show that there are two ways to i.e interquartile method and standard deviation.